build.gradle

```diff
@@ -171,8 +171,9 @@ dependencies {
     testCompile group: 'com.github.tomakehurst', name: 'wiremock-standalone', version: '2.21.0'
     testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.1.0.RELEASE'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.0'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.5_HMAN-368'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6-HMAN-368'
+
```

Comment:
These should be updated to match final version numbers if being merged to `master`.